packages/grafana-ui/src/components/Combobox/getComboboxStyles.ts

```diff
@@ -7,6 +7,7 @@ import { GrafanaTheme2 } from '@grafana/data';
 export const MENU_ITEM_FONT_SIZE = 14;
 export const MENU_ITEM_FONT_WEIGHT = 500;
 export const MENU_ITEM_PADDING_X = 8;
+export const MENU_ITEM_LINE_HEIGHT = 22;
```

Comment:
I think we should use a line high multiple (1.2, or something like that), and keep store the height calculation here once and have the const in styles, which is then referenced in useComboboxFloat.

Basically move OPTION_HEIGHT into this file.

Reply:
I prefer line height multiple because it makes the calculation more resilient to change if we adjust stuff like the font-size.

Reply:
We used `theme.typography.body.lineHeight` previously. Should we multiply `MENU_ITEM_FONT_SIZE` by 1.2 or similar? So if we change the `font-size` the height would adapt.

Reply:
Yeah, I think that would be ideal. I think that's what the lineHeight on theme object is now?

Reply:
Well, we [build the body variant](https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/themes/createTypography.ts#L115) setting `line-height` to 22, without calculation.

Reply:
Yeah, it's weird. It looks like we then divide that 22 by the font size to get the multiple back 

https://github.com/grafana/grafana/blob/af513964c8d3e9829f33f853625fac954c3c69a6/packages/grafana-data/src/themes/createTypography.ts#L101C7-L101C37